go.mod

```diff
@@ -20,13 +20,14 @@ require (
 	github.com/grpc-ecosystem/go-grpc-middleware v1.2.0
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
 	github.com/grpc-ecosystem/grpc-gateway v1.14.3
-	github.com/huandu/xstrings v1.3.0 // indirect
+	github.com/huandu/xstrings v1.3.2 // indirect
 	github.com/improbable-eng/grpc-web v0.12.0
 	github.com/jetstack/cert-manager v0.13.0
-	github.com/kubermatic/utils v0.0.0-20200707131849-09ebdec30295
```

Comment:
hmmm I don't see any tags in the utils repo. This should probably stay `v0.0.0-date-git_commit` format